src/TSTransformer/nodes/jsx/transformJsxTagName.ts

```diff
@@ -11,12 +10,17 @@ function transformJsxTagNameExpression(state: TransformState, node: ts.JsxTagNam
 	if (ts.isIdentifier(node)) {
 		const symbol = state.typeChecker.getSymbolAtLocation(node);
 		if (symbol) {
-			assert(state.services.roactSymbolManager);
-			const className = state.services.roactSymbolManager.getIntrinsicElementClassNameFromSymbol(symbol);
+			const className = state.services.jsxSymbolManager.getIntrinsicElementClassNameFromSymbol(symbol);
```

Comment:
Is this still necessary? It would appear to me that we can use only the latter check, and TS checks will ensure things are valid.

Reply:
So this is sorta necessary to maintain the mapping between `"textlabel"` -> `"TextLabel"`. But we _could_ maintain this mapping within Roact's package at runtime. Maybe that _is_ the better way to do this even if it is a little bit of duplication. It's certainly less "magic".